### DISCUSSION TITLE
phishing

### DISCUSSION THREAD

**Discussion by cyberturkeyyy** [community/community#144472] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

The reported URL address redirects to a phishing website. We kindly request your valuable assistance in closing the address.

https://github.com/ever2133-es/bets10-resmi
https://github.com/aarox4/bets10?tab=readme-ov-file

PLEASE NOTE THAT: THIS IS URL WORKING ONLY WITH ENTER TURKEY IP'S (TURKEY LOCATIONS USERS) IF YOU WILL ENTER WITH TURKEY LOCATION VPN SERVICE YOU WILL SEE PHISHING CONTENT

**Comment by Akash1134** [comment]
Hi @cyberturkeyyy , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
